src/go/rpk/pkg/cli/cmd/acl.go

```diff
@@ -65,3 +80,210 @@ func NewACLCommand(fs afero.Fs, mgr config.Manager) *cobra.Command {
 	command.AddCommand(acl.NewUserCommand(fs))
 	return command
 }
+
+const helpACLs = `Manage ACLs and SASL users.
+
+This command space creates, lists, and deletes ACLs, as well as creates SASL
+users. The help text below is specific to ACLs. To learn about SASL users,
+check the help text under the "user" command.
+
+When using SASL, ACLs allow or deny you access to certain requests. The
+"create", "delete", and "list" commands help you manage your ACLs.
+
+An ACL is made up of five components:
+
+  * a principal (the user)
+  * a host the principal runs on
+  * what resource to access (topic name, group ID, ...)
+  * the operation (read, write, ...)
+  * the permission: whether to allow or deny the above
+
+To specify a single ACL, you need to specify all five. All ACL commands combine
+the permission into the principal and host flags, meaning to specify a single
+ACL, you need four flags.
```

Comment:
This is a little redundant, given that above it says "`An ACL is made up of five components`".
```suggestion
All ACL commands combine the permission into the principal and host flags (i.e.
`--allow-principal`, `--deny-host`), meaning to specify a single ACL, you need to
pass at least four flags: `--operation`, `--resource`, `--{allow,deny}-principal`
and `--{allow,deny}-host`.
```

Reply:
Rereading this a few times, I think the paragraph can be deleted without any loss of clarity.